apps/src/blockly/TESTING.md

```diff
@@ -50,23 +50,23 @@ Create a new Poetry project: [[local_poetry]](http://localhost-studio.code.org:3
 Create a new Poetry HOC project: [[local_poetry_hoc]](http://localhost-studio.code.org:3000/projects/poetry_hoc/new) [[production_poetry_hoc]](https://studio.code.org/projects/poetry_hoc/new)
 - Check 2 versions of Poetry lab - poetry and poetry_hoc
 - Check shadow blocks, for example, 'set title' block. Most blocks that accept text have shadow blocks. You will have to remove an actual block that is covering the shadow block to see the shadow block.
-- Check variables - variable picker (purple block with '???') with customized options - look for custom modals to rename variables. 
+- Check variables - variable picker (purple block with '???') with customized options - look for custom modals to rename variables.
 - Under 'Effects', check the 'set background to' block. Its dropdown field is grid and includes a 'More' link. When 'More is clicked, modal opens to choose background to add to assets. Note that in July 2023, the 'More' button will change to 'Backgrounds'.
 - Check 'make new_sprite at_' block.
     - Click on the animation dropdown, then click on Costumes button which should take you to Costumes tab.
     - Click on location field, then confirm you can select new location of animation.
 
 
-# Project Beats / Music Lab
-Go to Project Beats: [[local]](http://localhost-studio.code.org:3000/projectbeats) [[production]](https://studio.code.org/projectbeats)
+# Music Lab
+Go to Music Lab: [[local]](http://localhost-studio.code.org:3000/projects/music/new) [[production]](https://studio.code.org/projects/music/new)
```

Comment:
super nit: reword to `"Create a new Music Lab project"` to match the others?

Reply:
Good catch.  Done.